### PR TITLE
Cookbook to install features from Windows SDK

### PR DESCRIPTION
**Pull request by jaym**
We'll need this to get the signing tools for signing MSIs in Omnibus

cc @adamedx @btm @ksubrama @schisamo @smurawski

### CODE REVIEW

**Comment by adamedx** [comment]
+1 on implementing as a resource and having single-purpose recipes.

File: libraries/windows_sdk_feature.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
+require_relative "./windows_sdk_helpers"
+class WindowsSdkCookbook
```
Comment (adamedx) [inline libraries/windows_sdk_feature.rb:2]: How do I configure installation location? I assume it's going to go to programfiles, which may not be the right place.

**Comment by adamedx** [comment]
We should consider implementing this using the resources cookbook, i.e. https://github.com/chef-cookbooks/resource.

**Comment by adamedx** [comment]
@jdmundrawala, I will re-implement https://github.com/adamedx/winbox/blob/master/recipes/debugger.rb to depend on this. Actually, I may just remove that recipe since it's not part of the default configuration of winbox anyway.

File: libraries/windows_sdk_feature.rb (hunk 1 above)
Comment (jaym) [inline libraries/windows_sdk_feature.rb:2]: will add an attribute

**Comment by usepowershell** [comment]
:+1:

**Pull request merged**